redaxo/src/core/lib/console/sync_assets.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @package redaxo\core
+ * 
+ * @auth staabm
+ *
+ * @internal
+ */
+class rex_command_sync_assets extends rex_console_command
+{
+    protected function configure()
+    {
+        $this
+            ->setDescription('Sync files of /assets with addons/assets core/assets folders');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = $this->getStyle($input, $output);
+
+        foreach(rex_package::getInstalledPackages() as $package) {
+            if ($package instanceof rex_addon) {
+                $feAssetsPath = rex_path::addonAssets($package->getName());
+                $beAssetsPath = rex_path::addon($package->getName(), 'assets/');
+            } else {
+                // we assume "plugin" assets folder is a subfolder of the "addon" assets folder
+                // and will therefore already be synced when handling the addon
+                continue;

```

Comment:
Kann der Überlegung nicht folgen. Die Plugin-Assets sind in `src/addons/foo/plugins/bar/assets`, müssen somit meiner Meinung nach separat gesynct werden.

Reply:
diese aussage bezieht sich auf `/assets/addons/meinAddon/plugins/meinPlugin/`  d.h. sobald der addon folder synchronisiert ist, sind automatisch auch die plugins darunter synchronisiert.